com.unity.render-pipelines.high-definition/Runtime/Debug/ProbeVolumeDebug.shader

```diff
@@ -65,7 +65,7 @@ Shader "Hidden/HDRP/ProbeVolumeDebug"
             float4 position = float4(UNITY_MATRIX_M._m03_m13_m23, 1);
             int brickSize = UNITY_ACCESS_INSTANCED_PROP(Props, _IndexInAtlas).w;
 
-            if (distance(position.xyz, _WorldSpaceCameraPos.xyz) > _CullDistance ||
+            if (distance(position.xyz, GetCurrentViewPosition()) > _CullDistance ||
```

Comment:
Uhm I am not sure I understand this. The position above should be absolute world position and as such we could compare with absolute position whereas this change will compare against 0 when camera relative is on.

Reply:
We can talk about this one when you're back. Was actually quite easy to repro.